tests/src/dir.targets

```diff
@@ -120,7 +119,8 @@
 
   <Target Name="CopyNativeProjectBinaries">
      <ItemGroup>
-        <NativeProjectBinaries Include="$(NativeProjectOutputFolder)\**\*.*" />
+        <NativeProjectBinaries Include="$(NativeProjectOutputFolder)\**\*.*" Condition="'$(BuildOS)' == 'Windows_NT'"/>
+        <NativeProjectBinaries Include="$(ProjectDir)\..\bin\Product\$(OSPlatformConfig)\bin\*.*" Condition="'$(BuildOS)' != 'Windows_NT'"/>
```

Comment:
that is fine for now, so please remove the above line and do the post build copy step and file a bug to track enabling it here.

if this is going to go unchanged, it will bloat the test bin folder.